odoo/addons/test_translation_import/tests/test_term_count.py

```diff
@@ -269,6 +269,72 @@ def test_export_pollution(self):
                 self.assertEqual(row['src'], "Accounting")
                 self.assertEqual(row['value'], "samva")
 
+    def test_translation_placeholder(self):
+        """Verify placeholder use in _()"""
+        self.env.ref("base.lang_fr_BE").active = True
+
+        # translation with positional placeholders
+        translation = self.env['ir.translation'].create({
+            'src': 'Text with %s placeholder',
+            'value': 'Text avec %s marqueur',
+            'type': 'code',
+            'name': 'addons/test_translation_import/tests/test_count_term.py',
+            'res_id': 0,
+            'lang': 'fr_BE',
+        })
+
+        context = {'lang': "fr_BE"}
+        # correctly translate
+        self.assertEqual(
+            _("Text with %s placeholder", 1),
+            "Text avec 1 marqueur",
+            "Translation placeholders were not applied"
+        )
+
+        # too many arguments should raise an error
```

Comment:
```suggestion
        # source error: wrong arguments
```
Too bad you cannot `assertNoLogs()` here :wink: